psycop/projects/cvd/model_training/preprocessing/regex_filter.py

```diff
@@ -0,0 +1,19 @@
+import polars as pl
+
+from psycop.common.model_training_v2.config.baseline_registry import BaselineRegistry
+from psycop.common.model_training_v2.trainer.preprocessing.step import (
+    PolarsFrame_T0,
+    PresplitStep,
+)
+
+
+@BaselineRegistry.preprocessing.register("regex_column_blacklist")
+class RegexColumnBlacklist(PresplitStep):
```

Comment:
@HLasse I've added a few modules here as 'hacks'. Let me know if you think any of them are interesting, then we should polish and add them to the shared code :-)

Reply:
(see the other files also)

Reply:
Same for you as well @sarakolding.

Reply:
I see myself using all of them within the next week's time so I propose moving them all of them to main :)